reporters/GlobalSummaryReporter.js

```diff
@@ -105,6 +109,11 @@ class GlobalSummaryReporter {
 
     // Pricefeed stats.
     this.priceEstimate = this.referencePriceFeed.getCurrentPrice();
+
+    // Initialize Uniswap pair to get trade data for. Default pair token is the collateral token.
+    this.uniswapPairToken = this.uniswapPairOverride[this.toChecksumAddress(this.empContract.options.address)]
```

Comment:
Reference for https://github.com/UMAprotocol/protocol/pull/1733/files#r452888913